doc/changelog.md

```diff
@@ -357,7 +396,7 @@
   nested branches. The `wires` property in `adjoint` and `ctrl` cannot be used in workflows with
   control flow operations.
 
-* Add support for lowering the eigen vectors/values computation lapack method: `lapack_dsyevd`
+* Add support for lowering the eigenvectors/values computation lapack method: `lapack_dsyevd`
   via `stablehlo.custom_call`. For Catalyst, it means that you now can QJIT compile eigen
   vector/values operations and any other `qml.math` methods that uses these operations.
   [(#488)](https://github.com/PennyLaneAI/catalyst/pull/488)
```

Comment:
@maliasadi I am a bit confused here, because `A` seems to be a NumPy array (and so `qml.math.sqrt_matrix()` would dispatch to `np.sqrt_matrix()`. That would imply it is run at tracing time, rather than being captured by JAX/JAXPR?